mgmtsystem_audit/data/audit_automated_actions.xml

```diff
@@ -33,7 +34,7 @@ Odoo
         <field name="model_id" ref="model_mgmtsystem_audit" />
         <field name="template_id" ref="template_email_prepare_to_audit" />
         <field name="trigger">on_time</field>
-        <field name="state">email</field>
+        <!-- <field name="state">email</field> -->
```

Comment:
```xml
<field name="state">mail_post</field>
```

I think you want `mail_post` here, no?
Commenting it out will fallback to the default `object_write`, and that's not what we want here

See: https://github.com/odoo/odoo/commit/b3fc41cc44b251be37b6aaf2931179afeb126d69

Reply:
(same for the other case, btw)

Reply:
@ivantodorovich 
Thanks.